module/zfs/vdev_raidz_math.c

```diff
@@ -459,18 +463,10 @@ vdev_raidz_math_init(void)
 	membar_producer();		/* complete raidz_supp_impl[] init */
 	raidz_supp_impl_cnt = c;	/* number of supported impl */
 
-#if !defined(_KERNEL)
-	/* Skip benchmarking and use last implementation as fastest */
-	memcpy(&vdev_raidz_fastest_impl, raidz_supp_impl[raidz_supp_impl_cnt-1],
-	    sizeof (vdev_raidz_fastest_impl));
-	strcpy(vdev_raidz_fastest_impl.name, "fastest");
-
-	raidz_math_initialized = B_TRUE;
```

Comment:
Missing raidz_math_initialized = B_TRUE in new code.

Reply:
This function was reworked slightly to remove the early return.  It now falls through and sets `raidz_math_initialized` at the end of the function in a single location.  Additionally, the logic to default to "cycle" was moved in to `ztest` as a more appropriate home.